docs/blueprints/EP038.rst

```diff
@@ -0,0 +1,186 @@
+
+By leveraging the power of aio-pika, we can seamlessly integrate RabbitMQ's robust messaging capabilities into our Kytos Napps. This potent combination of RabbitMQ's strength and aio-pika's asynchronous nature inspires the creation of scalable and responsive distributed systems, fueling our projects' potential.
+
+In conclusion, isolating the Broadcasting Events Napp and the Events Consumer Napp into separate microservices from the Message Queue Producer and Consumer Napps promotes flexibility, modularity, scalability, and resilience in the system's architecture. This design approach enables the system to adapt to changing requirements and technologies while maintaining robustness and efficiency in event broadcasting and consumption.
+
+Use Case: Interdomain Link Up/Down Monitoring
+
+Scenario: In SDX, monitoring the status of interdomain links for connectivity and reliability is crucial. This use case involves setting up a system to detect link status changes (up/down) and notify consumers about these events through message queues.
+
+Components:
+Producer: Generates events based on link status changes.
+Consumer: Monitors link status by consuming messages from the appropriate queues.
+
+Implementation:
+    
+Link Status Queue Initialization:
+    
+Each interdomain link has its dedicated queue.
+Queues are either dynamically created or configured based on settings.
+
+Benefits:
+Real-time monitoring of interdomain link status.
+Scalable solution with dynamically created queues.
+Fault-tolerant design ensures persistent handling of link-down events.
+Flexibility in queue management allows dynamic addition or configuration based on settings.
+
+With its efficient producer-consumer relationship, the system orchestrates message handling for specific link-down events. This design ensures smooth communication, preventing message flooding and instilling confidence in its performance. Here's how it operates:
+
+Producer Advertises Link Down: When a link-down event occurs, the producer publishes a message indicating the link's status change to down.
+
+Consumer's Crucial Role in Link Status Monitoring: The consumer, a key player, diligently monitors the message queue for link-down events. However, if the consumer has not yet read the message or the link status hasn't changed, the producer refrains from continuously writing messages of the same status to the queue.
+
+Preventing Message Flooding: To prevent message flooding and conserve system resources, the producer only writes messages to the queue if the consumer has consumed the previous message or the link status has changed.
+
+Message continuity is based on consumer activity; if the consumer has not acknowledged or processed the previous message, the producer waits for them to read it before publishing another message with the same status. In the same way, if the link status changes, the producer updates the message accordingly. 
```

Comment:
> the producer waits for them to read it before publishing another message with the same status. In the same way, if the link status changes, the producer updates the message accordingly.

This is coupling producer with a consumer. It sounds strange to me. I've never seen any type of pub sub where a publisher were to do it. It's also contradicting some of the prior information about producer being independent from a consumer (as it should be). Can you double check this paragraph?